src/pytype.jl

```diff
@@ -261,7 +261,7 @@ type PyTypeObject
     # Julia-specific fields, after the end of the Python structure:
 
     # save the tp_name Julia string so that it is not garbage-collected
-    tp_name_save::ASCIIString
+    tp_name_save # This is a pointer field that is never read from
```

Comment:
This is not correct.  It is not a raw pointer field, it is a Julia object, stored to prevent garbage-collection.